demo/cms/render/react-renderer.jsx

```diff
@@ -12,7 +12,7 @@ const assetManifest = {
 	'main.js': 'main.js',
 };
 
-const ASSET_ROOT = 'http://local.in.ft.com:8080/statics/';
+const ASSET_ROOT = '/';
```

Comment:
does this still work when running the demo app locally and outside of cypress i.e. `make run`?

Reply:
do we even need webpack dev server any more? I'm happy to remove it

Reply:
yes, it does work